config/archive_test.go

```diff
@@ -0,0 +1,120 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/contiv/volplugin/watch"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	testPolicyName = "foobar"
+)
+
+func revisionCount(s *configSuite) int {
+	revisions, _ := s.tlc.ListPolicyRevisions(testPolicyName)
```

Comment:
"Does not exist" errors are raised here in a few cases, e.g., when verifying that a policy has zero revisions, if the policy hasn't been uploaded yet there is no directory, so etcdclient returns an `errored.Error`.  

I see some assertions which check that the error text matches some expected value.  I could do that, or I could just defer/recover and return 0.

What is the idiomatic way to handle this?